examples/microwave-oven-app/microwave-oven-common/microwave-oven-app.zap

```diff
@@ -3183,6 +3183,22 @@
           "attributes": [
             {
               "name": "CookTime",
+              "code": 0,
+              "mfgCode": null,
+              "side": "server",
+              "type": "elapsed_s",
+              "included": 1,
+              "storageOption": "External",
+              "singleton": 0,
+              "bounded": 0,
+              "defaultValue": null,
+              "reportable": 1,
+              "minInterval": 1,
+              "maxInterval": 65534,
```

Comment:
Should this match the max allowed in the spec?  If so the other cases of maxInterval should also be changed.
```suggestion
              "maxInterval": 86400,
```

Reply:
Seems it is casued by the type is belongs to elapsed_s, the max value of elapsed_s is 65534.

But I have also set the max value  to 86400 in the XML file, it dose not work...

Currently we set the defalut value in the SDK code, while the value from ZAP file is  not used anymore, could we ignore the ZAP config value on this field?

Reply:
@rbultman I try to change the `maxInterval` to 86400 in the ZAP config however is not allowed, the reason is out of range. Seems no need to change it.